pkgs/development/python-modules/cfv/default.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+}:
+
+buildPythonPackage rec {
+  pname = "cfv";
+  version = "3.1.0";
+
+  src = fetchFromGitHub {
+    owner = "cfv-project";
+    repo = "cfv";
+    rev = "refs/tags/v${version}";
+    sha256 = "1wxf30gsijsdvhv5scgkq0bqi8qi4dgs9dwppdrca5wxgy7a8sn5";
+  };
+
+  meta = {
+    description = "cfv is a utility to test and create a wide range of checksum verification files";
```

```suggestion
    description = "Utility to verify and create a wide range of checksums";
```
Description should not repeat package name.